oap-server/server-configuration/configuration-k8s-configmap/src/main/java/org/apache/skywalking/oap/server/configuration/configmap/ConfigmapConfigurationWatcherRegister.java

```diff
@@ -46,9 +45,7 @@ public ConfigmapConfigurationWatcherRegister(ConfigmapConfigurationSettings sett
             if (log.isDebugEnabled()) {
                 log.debug("read config: name:{} ,value:{}", name, value);
             }
-            if (Objects.nonNull(value)) {
-                configTable.add(new ConfigTable.ConfigItem(name, value));
-            }
+            configTable.add(new ConfigTable.ConfigItem(name, value));
```

Comment:
This should be documented as a bug fix in the changelog.

Reply:
noted